tests/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-find_package(Boost 1.55.0 REQUIRED)
+# We move the test files into the testing directory
+file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/hypervolume_test_data/" DESTINATION "${CMAKE_BINARY_DIR}/tests/hypervolume_test_data/")
```

Comment:
That's rather weird. Could you try to do a debug commit on appveyor with the printing code in cmake to see what it outputs?